### PR TITLE
Version bump: xfce-extra/multiload-nandhp-0.3

### PR DESCRIPTION
**Pull request by j123b567**
```
- fix #527032
- add Czech translation
- remove lxpanel support (lxde-base/lxpanel-0.7 and newer are not supported by upstream becuase of API change in lxpanel)
```

### CODE REVIEW

File: xfce-extra/multiload-nandhp/multiload-nandhp-0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+DESCRIPTION="A port of the GNOME multiload applet for the Xfce panel"
+HOMEPAGE="http://github.com/nandhp/multiload-nandhp"
+SRC_URI="http://github.com/nandhp/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE=""
+
+RDEPEND=">=x11-libs/gtk+-2.14:2
```
Comment (mgorny) [inline xfce-extra/multiload-nandhp/multiload-nandhp-0.3.ebuild]: we could do `2=` here just to be sure.

File: xfce-extra/multiload-nandhp/multiload-nandhp-0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+DOCS="AUTHORS README.md"
+
+S=${WORKDIR}/${PN}-${MY_PV}
+
+src_prepare() {
+	eautoreconf
+}
+
+src_configure() {
+	econf xfce4
```
Comment (mgorny) [inline xfce-extra/multiload-nandhp/multiload-nandhp-0.3.ebuild]: This isn't correct. `--with-xfce4`, I guess.

Comment (j123b567) [inline xfce-extra/multiload-nandhp/multiload-nandhp-0.3.ebuild]: Beginner's mistake. You are of course right.

File: xfce-extra/multiload-nandhp/multiload-nandhp-0.2.ebuild [hunk 3]
```diff
@@ -1,52 +0,0 @@
...
-
-MY_PV=version-${PV/./-}
-
-DESCRIPTION="A port of the GNOME multiload applet for the Xfce and LXDE panels"
-HOMEPAGE="http://github.com/nandhp/multiload-nandhp"
-SRC_URI="http://github.com/nandhp/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
-
-LICENSE="GPL-2"
-SLOT="0"
-KEYWORDS="~amd64 x86"
```
Comment (mgorny) [inline xfce-extra/multiload-nandhp/multiload-nandhp-0.2.ebuild:16]: The old version is stable on `x86`, so you can't just remove it without getting the newer version tested and stabilized first. And anyway, it's often a good idea to delay the removal of old version after adding the new one.

File: xfce-extra/multiload-nandhp/multiload-nandhp-0.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+MY_PV=version-${PV/./-}
+
+DESCRIPTION="A port of the GNOME multiload applet for the Xfce panel"
+HOMEPAGE="http://github.com/nandhp/multiload-nandhp"
+SRC_URI="http://github.com/nandhp/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (mgorny) [inline xfce-extra/multiload-nandhp/multiload-nandhp-0.3.ebuild]: You can't add a new version straight to stable. `~amd64 ~x86`.

File: xfce-extra/multiload-nandhp/metadata.xml [hunk 5]
```diff
@@ -2,9 +2,6 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
   <herd>xfce</herd>
-  <use>
-    <flag name="lxpanel">Build the applet for <pkg>lxde-base/lxpanel</pkg>, you can set XFCE_PLUGINS="" if you don't want the Xfce dependencies.</flag>
```
Comment (mgorny) [inline xfce-extra/multiload-nandhp/metadata.xml:6]: Remember to restore this along with the old version.

**Comment by mgorny** [comment]
And _please_ always run `repoman manifest` and `repoman full` after changes, and see if no new errors/warnings appear.

**Comment by j123b567** [comment]
I always run `repoman full` but before `push`, I must run it before `commit`, because behaviour is different.

**Comment by mgorny** [comment]
```
+*multiload-nandhp-0.3 (13 Jul 2015)
+
+  13 Jul 2015; Michał Górny <mgorny@gentoo.org> +multiload-nandhp-0.3.ebuild:
+  Version bump. Adds Czech translation, fixes missing POTFILES.in (bug #527032).
+  lxpanel plugin has been disabled because of incompatible API change.
+  https://github.com/gentoo/gentoo-portage-rsync-mirror/pull/161 by j123b567.
```

**Pull request closed without merging**